webservices/docs.py

```diff
@@ -392,6 +392,9 @@
         - X party, nonqualified\n\
         - Y party, qualified\n\
 '
+COMMITTEE_LABEL = '''
+Displayed committee type
```

Comment:
@hcaofec Can you change the `committee_label` description to `Committee name based  on cmte_type, cmte_dsgn and org_tp`?